src/libpmemfile/preload.c

```diff
@@ -229,6 +231,40 @@ log_write(const char *fmt, ...)
 	syscall_no_intercept(SYS_write, log_fd, buf, len);
 }
 
+void
+exit_group_no_intercept(int ret, const char *msg)
+{
+	if (msg) {
+		if (msg[0] == '!') {
+			syscall_no_intercept(SYS_write, 2, msg + 1,
```

Comment:
Fair point, I can remove strerror_r, but I'm not sure what to do about snprintf. I'd like to print errno somehow. Ideas?